docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,79 +1,96 @@
 
-## Step by step instructions
+**1. Add the "OpenProject integration" app and connect the instance** - in Nextcloud
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly out of their Nextcloud instance.
+To activate your integration to OpenProject in Nextcloud, navigate to the built-in app store in the menu under your user name by clicking on _Your apps_. You can use the search field in the top right corner to look for the OpenProject integration app. Click the button **Download and enable**.
 
-**Add OpenProject integration app**
+![Nextcloud_app_store](Nextcloud_app_store.png)
 
-To activate your integration to OpenProject in Nextcloud, navigate to the built in app store under your user name in Your apps. You can use the search field in the top right corner to look for the OpenProject integration. Click the button Download and enable.
+Once the OpenProject integration app is downloaded and enabled, you can access it from within the Settings page, via the the side menu. Start by entering the *host URL* of your desired OpenProject instance.
 
-![Nextcloud_app_store](Nextcloud_app_store.png)
+![Nextcloud_host](3_2_01 -NC_Step_1.png)
+
+Click on the **Save** button and open your OpenProject instance in a new tab.
+
+**2. Introduce the basic information of your integration** - in OpenProject
+
+Make sure you are logged in as an administrator in your OpenProject instance. Navigate to *File storages* from the Administration page. To start configuring your new Nextcloud integration, click on the **Add (+)** button to add a new file storage.
+
+![Storage_index](3_0_00-OP_OAuth_Empty_Index.png)
+
+By default, the storage *provider type* is set to Nextcloud; this does not need to be modified. To proceed, enter a *name* to your Nextcloud storage. This will be visible to all the users using it. We highly recommended choosing a clear and distinct name that allows users to differentiate it from other potential file storages integrations in the future. You will also need to set the *Host URL* of your Nextcloud instance, including _https://_ bit of the address.
+
+![Storage_basic_information](3_0_01-OP_General_Info.png)
+
+Once you fill in the mandatory fields, click on the **Save and continue setup** button.
+
+**3. Generate and copy the OpenProject OAuth values** - in OpenProject and Nextcloud
+
+In this step, the *OpenProject OAuth values* are generated automatically. These values are needed to permit OpenProject to connect to Nextcloud. You will need to copy them from the Nextcloud integration window in a new tab (without exiting or closing your OpenProject tab) and pasting them in the respective fields under _OpenProject OAuth settings_.
```

Comment:
```suggestion
In this step, the *OpenProject OAuth values* are generated automatically. These values are needed to permit OpenProject to connect to Nextcloud. If you have not done so yet, we recommend you to best have two browser tabs open, the current one with OpenProject and the former one with Nextcloud. Now copy the two *OpenProject OAuth values* from the OpenProject tab to the respective fields in the Nextcloud tab. Simply paste them into the respective _OpenProject OAuth settings_ fields.
```

Reply:
"We recommend you have" (vs. "to best have").